src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        return new Ball();
```

Comment:
logic of creation the Random Ball should be here. 

Ball.class shouldn't consist of random logic creation.

Reply:
Didn't commit my changes, sorry